.github/workflows/metadata_service_deploy_orchestrator_dagger.yml

```diff
@@ -27,3 +32,19 @@ jobs:
           gcp_gsm_credentials: ${{ secrets.GCP_GSM_CREDENTIALS }}
         env:
           DAGSTER_CLOUD_METADATA_API_TOKEN: ${{ secrets.DAGSTER_CLOUD_METADATA_API_TOKEN }}
+          DAGSTER_CLOUD_DEPLOYMENT: "prod"
+      - name: Deploy the metadata orchestrator [On workflow]
+        id: metadata-orchestrator-deploy-orchestrator-pipeline-branch
+        if: github.event_name == 'workflow_dispatch'
+        uses: ./.github/actions/run-airbyte-ci
+        with:
+          subcommand: "metadata deploy orchestrator"
+          context: "manual"
+          dagger_cloud_token: ${{ secrets.DAGGER_CLOUD_TOKEN_2 }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          docker_hub_username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          docker_hub_password: ${{ secrets.DOCKER_HUB_PASSWORD }}
+          gcp_gsm_credentials: ${{ secrets.GCP_GSM_CREDENTIALS }}
+        env:
+          DAGSTER_CLOUD_METADATA_API_TOKEN: ${{ secrets.DAGSTER_CLOUD_DEV_METADATA_API_TOKEN }}
```

Comment:
Hmm do we need to select the API token according to the deployment target?

Reply:
So I added this just to have a separation.

But in reality, the dev token can access prod, and prod can access the dev token because Dagster scopes them to the user, and all users and full access to both with no way to granularly change that

That will likely change at some point in the future so this is like a preprepreoptomization🤷

Reply:
That makes sense based on my experience. I think the pre-optimization might be more confusing!

Reply:
Ill change :)